controllers/apply/form-router-next/steps.js

```diff
@@ -237,12 +237,24 @@ module.exports = function(formId, formBuilder) {
                         }
                     }
 
-                    const { nextPage } = form.pagination({
-                        baseUrl: res.locals.formBaseUrl,
-                        sectionSlug: req.params.section,
-                        currentStepIndex: stepIndex,
-                        copy: res.locals.copy
-                    });
+                    let redirectUrl;
+                    if (req.body.previousBtn) {
+                        const { previousPage } = form.pagination({
+                            baseUrl: res.locals.formBaseUrl,
+                            sectionSlug: req.params.section,
+                            currentStepIndex: stepIndex,
+                            copy: res.locals.copy
+                        });
+                        redirectUrl = previousPage.url;
+                    } else {
+                        const { nextPage } = form.pagination({
```

Comment:
It would be nice to turn this into a function somewhere since it's basically the same code for `nextPage`/`previousPage`.

Reply:
Sure. I'll put this in a named function. Will also add the `cursor: pointer` to the .btn-link